lib/cell.rb

```diff
@@ -0,0 +1,11 @@
+class Cell
+  attr_reader :symbol
+
+  def set_symbol(symbol)
+    @symbol = symbol
+  end
+
+  def empty?()
+    symbol == nil
```

Comment:
```suggestion
    symbol.nil?
```
Also a suggestion made by @ssunday